man/org.freedesktop.home1.xml

```diff
@@ -358,9 +363,25 @@ node /org/freedesktop/home1 {
       triggered deactivation is completed. This method is equivalent to <function>Release()</function> on the
       <classname>org.freedesktop.home1.Home</classname> interface.</para>
 
+      <para><function>InhibitSuspendHome()</function> temporarily inhibits automatic locking during system
+      suspend for a home directory. It returns a file descriptor that inhibits this functionality for as long
+      as it is open. As mentioned above, locking a home directory requires a secure lock screen running
+      outside of the user context, and is likely to freeze any process that attempts to access the directory.
+      Thus, locking a home directory is a trade-off: it increases security, but prevents the client from
+      displaying any user content on its secure lock screen, including notifications, media controls, contact
+      information for incoming phone calls, and much more. A client may use this method to implement more
+      complicated automatic locking behavior for home directories, in order to solve some of these UX issues.
+      For instance, the client may choose to only lock the home directory and switch to the secure lock screen
+      if the device has been suspended for over 24 hours. Note that this inhibitor does not prevent clients from
```

Comment:
how would they implement the 24h thing though?

Reply:
Immediately before suspend, start a .timer unit that will wake the device up in 24 hours. If the device is awoken before the timer fires, stop the timer. If the device is awoken by the timer, then lock homed, switch to the secure GDM lock screen, and put the device back to sleep.